stake-pool/cli/src/main.rs

```diff
@@ -1232,12 +1283,32 @@ fn main() {
                     .help("Public key for the new account to set as the stake pool fee receiver."),
             )
             .group(ArgGroup::with_name("new_accounts")
-                .arg("new_owner")
+                .arg("new_manager")
                 .arg("new_fee_receiver")
                 .required(true)
                 .multiple(true)
             )
         )
+        .subcommand(SubCommand::with_name("set-staker")
+            .about("Change staker account for the stake pool. Must be signed by the manager.")
```

Comment:
It would be nice if the current staker could change the staker as well.  This is the case for stake accounts as well.  The use case is that you don't want to bother your boss (the manager), they already delegated you the authority